src/quick-lint-js/error.h

```diff
@@ -357,6 +357,12 @@
       { source_code_span expected_left_curly; },                               \
       .error(QLJS_TRANSLATABLE("expected '{{'"), expected_left_curly))         \
                                                                                \
+  QLJS_ERROR_TYPE(                                                             \
+      error_comma_not_allowed_between_class_methods, "E209",                   \
```

Comment:
Nit: Sort errors alphabetically by struct name. This one would go around line 225.

Reply:
Moved to sorted line